### ISSUE TITLE
DAX Script flags measures with brackets in the name as "Syntax Error"

### ISSUE THREAD

**Issue by atmisra** [TabularEditor/TabularEditor3#239]
Perhaps not best practice to have brackets in the name of a measure, I know = ). But the DAX Script will consistently flag these measures as having a syntax error despite being syntactically correct. Biggest issue is that DAX script will not execute until it sees the error cleared. Simple example below.

DAX Script Output:
-----------------------------------------
-- Measure: [demo_measure [with_bracket_text]]
-----------------------------------------
MEASURE 'measuretable'[demo_measure [with_bracket_text]] = 123=456

Error:
Syntax Error near ]. Expected =

**Comment by otykier** [comment]
The issue is that the generated code does not correctly escape the bracket. Whenever you're referring to a measure or column whose name contains a right-bracket `]` you have to escape it by typing it twice: `]]`.

So in other words, if you change the code to:

```dax
MEASURE 'measuretable'[demo_measure [with_bracket_text]]] = 123=456
```

Then you should no longer see the error message. We will make sure to fix the DAX script code generation to escape the bracket correctly (same issue exists when you drag and drop a measure from the TOM Explorer into a DAX expression)

**Comment by otykier** [comment]
Fixed in [3.1.4](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.1.4). Thanks for reporting!
